WalletWasabi/Services/SystemAwakeChecker.cs

```diff
@@ -101,5 +101,36 @@ protected override async Task ActionAsync(CancellationToken cancel)
 				}
 			}
 		}
+
+		public void PreventShutdown()
+		{
+		}
+
+		public void PreventSleep()
+		{
+		}
+
+		public void ReleaseAllPrevention()
+		{
+		}
+
+		public void Update(WalletManager walletManager)
```

Comment:
`SystemAwakeChecker.cs` has its own `WalletManager` property. I'm not sure if we need to send it in as a param.

Reply:
ACK!